config/environments/development.rb

```diff
@@ -40,4 +40,14 @@
 
 
   config.action_mailer.default_url_options = { host: 'thingr.local' }
+
+  config.paperclip_defaults = {
+    :storage => :fog,
+    :fog_credentials => {
+      :provider => "AWS",
+      :aws_access_key_id => ENV['AWS_ACCESS_KEY_ID'],
+      :aws_secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+    },
+      :fog_directory => ENV['S3_BUCKET_NAME'],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.